test/v2/config/environment/broken/preprod.json

```diff
@@ -6,7 +6,7 @@
   "EthRPCURLs": [
     "https://ethereum-holesky-rpc.publicnode.com"
   ],
-  "BLSOperatorStateRetrieverAddr":          "0x003497Dd77E5B73C40e8aCbB562C8bb0410320E7",
+  "OperatorStateRetrieverAddr":          "0x003497Dd77E5B73C40e8aCbB562C8bb0410320E7",
```

Comment:
@cody-littley changed your config names. Prob will require you to update devops repo next time you want to deploy. Are you ok with this?

Reply:
I'm ok with this. I might just go ahead and convert the traffic generator to utilize the contract directory so we can be done with this flag. 😄